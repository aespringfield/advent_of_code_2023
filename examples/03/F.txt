234*45.....84.....5
...........*.....*.
83.....*..123.....6
*.............98...
890............*89.
