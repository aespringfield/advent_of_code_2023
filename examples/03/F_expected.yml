part_two: 103_484
